scripts/stats/index.js

```diff
@@ -110,9 +110,12 @@ export async function run() {
     (await countCards(COLUMN_ID_RFCS_ACCEPTED)).length + (await countCards(COLUMN_ID_RFCS_PRIORITIZED)).length,
     // Date (ISO)
     `"${new Date().toISOString()}"`,
-  ];
+  ].join(',');
 
-  appendFileSync('scripts/stats/stats.csv', entry.join(',') + '\n');
+  const statCsv = readFileSync('scripts/stats/stats.csv', {encoding: 'utf-8'});
+  const [statHeader, ...statItems] = statCsv.split('\n');
+  const updatedStatCsv = [statHeader, entry, ...statItems].join('\n');
+  writeFileSync('scripts/stats/stats.csv', updatedStatCsv);
```

Comment:
instead of a dumb file append, we now read the file first so that we can inject the new line directly into the document, before overwriting it back to disk.